apps/web/screens/Project/Project.tsx

```diff
@@ -100,6 +107,23 @@ const ProjectPage: Screen<PageProps> = ({ projectPage, news, namespace }) => {
     return x.slug === router.query.subSlug
   })
 
+  const navigationList: NavigationItem[] = projectPage.sidebarLinks.map(
+    ({ primaryLink, childrenLinks }) => ({
+      title: primaryLink.text,
+      href: primaryLink.url,
+      active: router.asPath === primaryLink.url,
+      items: childrenLinks.map(({ text, url }) => ({
+        title: text,
+        href: url,
+      })),
+    }),
+  )
+
+  const navigationData: NavigationData = {
+    title: n('navigationTitle', 'Efnisyfirlit'),
```

Comment:
is `activeItemTitle` never set to something?

Reply:
Good catch. It's never set, but we should be setting it, in order to improve the UX on mobile. We'll fix this.